LAUNCH.md

```diff
@@ -0,0 +1,200 @@
+# 🤝 Gflownet Launch tool help
+
+## 💻 Command-line help
+
+```sh
+usage: launch.py [-h] [--help-md] [--job_name JOB_NAME] [--outdir OUTDIR]
+                 [--cpus_per_task CPUS_PER_TASK] [--mem MEM] [--gres GRES]
+                 [--partition PARTITION] [--modules MODULES]
+                 [--conda_env CONDA_ENV] [--venv VENV] [--code_dir CODE_DIR]
+                 [--jobs JOBS] [--dev] [--verbose] [--force]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --help-md             Show an extended help message as markdown. Can be
+                        useful to overwrite LAUNCH.md with `$ python launch.py
+                        --help-md > LAUNCH.md`
+  --job_name JOB_NAME   slurm job name to show in squeue. Defaults to crystal-
+                        gfn
+  --outdir OUTDIR       where to write the slurm .out file. Defaults to
+                        $SCRATCH/crystals/logs/slurm
+  --cpus_per_task CPUS_PER_TASK
+                        number of cpus per SLURM task. Defaults to 2
+  --mem MEM             memory per node (e.g. 32G). Defaults to 32G
+  --gres GRES           gres per node (e.g. gpu:1). Defaults to gpu:1
+  --partition PARTITION
+                        slurm partition to use for the job. Defaults to long
+  --modules MODULES     string after 'module load'. Defaults to anaconda/3
+                        cuda/11.3
+  --conda_env CONDA_ENV
+                        conda environment name. Defaults to gflownet
+  --venv VENV           path to venv (without bin/activate). Defaults to None
+  --code_dir CODE_DIR   cd before running main.py (defaults to here). Defaults
+                        to ~/ocp-project/gflownet
+  --jobs JOBS           run file name in external/jobs (without .yaml).
```

Comment:
Why without `.yaml`? I find that a bit confusing. It makes me think - when I read the examples in the help text - that I have to pass a path, instead of a file.

Reply:
I'll change the docstring, it's misleading: you _can_ give a `.yaml` extension. Or not. It's usually just shorter to avoid adding it but it'll work just fine with it

Reply:
Ah perfect then :+1: